release/models/acl/openconfig-packet-match.yang

```diff
@@ -324,9 +330,15 @@ module openconfig-packet-match {
     }
 
     leaf destination-address {
-      type oc-inet:ipv4-prefix;
+      type oc-inet:ipv4-address;
```

Comment:
Yeah, let's not do this. This is a breaking change that will cause all consumers to need to change immediately.

If we really think that this prefix specification is not needed, then we should mark this as `deprecated` and then define a new leaf. Otherwise, we should consider adding new leaves for the specific case that is of concern.

It seems like there is an ask for being able to match a prefix against some bitmask. I'd consider this somewhat of a special case and say that we should probably have something like a `masked-source-address` and `source-address-mask` leaf (and corresponding destination based value).

I'd suggest we _only_ use these leaves in the case of non-wildcard matches that cannot be expressed by a CIDR mask.